scripts/cord19/temporal_analysis.py

```diff
@@ -0,0 +1,93 @@
+#
+# Pyserini: Python interface to the Anserini IR toolkit built on Lucene
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import argparse
+import pandas as pd
+import matplotlib.pyplot as plt
+
+sys.path.insert(0, './')
+
+from pyserini.collection import Collection, Cord19Article
+
+
+def main(path):
+    empty_date = dict()
+    normal_dates = dict()
+
+    cnt = 0
+    collection = Collection('Cord19AbstractCollection', path)
+    articles = collection.__next__()
+
+    #interate through raw collection
+    for (i, d) in enumerate(articles):
+        article = Cord19Article(d.raw)
+        # documents with empty abstract
+        metadata = article.metadata()
+        date = metadata['publish_time']
```

Comment:
can you document the fields of the df and their semantics in a comment?

Reply:
do you mean what fields do the metadata have or the other df I created for plotting?

Reply:
the df you create